packages/vanilla-i18n/src/translationStore.ts

```diff
@@ -41,6 +59,11 @@ export function translate(str: string, defaultTranslation?: string): string {
         return translationStore[str];
     }
 
+    // report any untranslated strings when in translation debug mode
+    if (defaultTranslation === undefined && translationDebug(getMeta("context.translationDebug"))) {
```

Comment:
Please use the `internalTranslationDebugValue` here. These packages should never have a dependency on library.

Reply:
removed the getMeta call